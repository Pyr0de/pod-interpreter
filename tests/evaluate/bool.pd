1+3==4;
false && false != true;
true + true;
true || false;
true && false;
!true;
!false;
