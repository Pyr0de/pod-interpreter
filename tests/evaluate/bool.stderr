ExitCode: 66
Cannot operator on "true + true"
